chainercv/transforms/bbox/translate_bbox.py

```diff
@@ -0,0 +1,27 @@
+def translate_bbox(bbox, x_offset=0, y_offset=0):
+    """Translate bounding boxes.
+
```

Comment:
How about adding a description on the usage?

This is used together with image transforms, such as padding and cropping, which translates the left top point of the image from coordinate :math:`(0, 0)` to coordinate :math:`(x_offset, y_offset)`.